libs/SalesforceSDK/src/com/salesforce/androidsdk/app/SalesforceSDKManager.java

```diff
@@ -454,9 +454,6 @@ public static void initInternal(Context context) {
         // Upgrades to the latest version.
         SalesforceSDKUpgradeManager.getInstance().upgrade();
 
-        // Initializes the encryption module.
-        Encryptor.init(context);
```

Comment:
This made sense in the `Eclair` and `Froyo` days, but I don't think it's required anymore.